daffodil-test/src/test/resources/org/apache/daffodil/section05/simple_types/SimpleTypes.tdml

```diff
@@ -5935,12 +5935,13 @@
 -->
   
   <tdml:parserTestCase name="dateCalendarDaysInFirstWeek5" root="date20"
-    model="dateTimeSchema" description="Section 13 Simple Types - calendarDaysInFirstWeek - DFDL-13-143R">
+    model="dateTimeSchema" description="Section 13 Simple Types - calendarDaysInFirstWeek - DFDL-13-143R"
+    roundTrip="twoPass">
```

Comment:
So what I believe is actually happening is that on parse  "1 03 2013" with dfdl:calendarPattern="W MM yyyy" gets converted to 2013-02-24, but when we go to unparse that,  ICU converts it to "5 02 2013" (read as last week of February), which is functionally equivalent to "1 03 2013" (read as first week of March).  Note that in 2013 the last week of February also contains the first 2 days of March.

So really I believe the data is just getting normalized to a different value than the original input. Both values are valid.  Another option instead of making this a twoPass test is to just change the input data to "5 02 2013".

Reply:
Just insert comment to the effect of what you just said as for why this is two pass. I think this is a good policy for twoPass tests, ... always comment on why they have to be two pass.